### DISCUSSION TITLE
7.0.0 Cleanup

### DISCUSSION THREAD

**Discussion by aucampia** [RDFLib/rdflib#1628] [TO-DO list]
There are many small things I think should be cleaned up for 7.0.0, not sure where to track these things, but I figured this may be as good a place as any. Alternatively an issue could also work but they are all quite small and will just make issue spam.

I made a comment on this conversation with a the list of things I want to clean up and I will keep adding to it, feedback to my list can be given as replies to that comment that. Others are welcome to make similar lists on this conversation.

**Comment by aucampia** [comment]
1. [ ] remove `DEBUG` from `query` methods in `rdflib/plugins/stores/sparqlstore.py` (added: 2021-12-28)
2. [ ] rename private functions/methods/variables/attributes/packages to start with underscore (added: 2021-12-28) (maybe just mark non underscore variants as deprecated in 7.0.0)
3. [ ] change `TypeError` for missing methods to `NotImplemented` in `rdflib/plugins/stores/sparqlstore.py` (added: 2021-12-28)
4. [ ] change `__repr__` to do what [`__repr__` should](https://docs.python.org/3/reference/datamodel.html#object.__repr) https://github.com/RDFLib/rdflib/blob/43d86224382d64fba2e5e17acedea1704631e097/rdflib/extras/infixowl.py#L2007
5. [ ] remove [`rdflib/extras/cmdlineutils.py`](https://github.com/RDFLib/rdflib/blob/master/rdflib/extras/cmdlineutils.py)

**Comment by nicholascar** [comment]
* [ ] Link Store listing to Persistence in docs
* [ ] Document parsing data into `Datasets` & `ConjunctiveGraphs`
* [ ] Ensure all supported Parsers & Serializers are listed correctly in docs

**Comment by ghost** [comment]
Re (2): Some materiel ...
1. test showing [basic ops on Dataset](https://github.com/gjhiggins/rdflib/blob/identifier-as-context/test/test_identifier_as_context/test_dataset_default_graph.py)
2. test showing the [right way to add a Graph](https://github.com/gjhiggins/rdflib/blob/identifier-as-context/test/test_identifier_as_context/test_dataset_example_use_pattern.py)
3. test showing [SPARQL UPDATE ops on Dataset](https://github.com/gjhiggins/rdflib/blob/identifier-as-context/test/test_identifier_as_context/test_dataset_sparql_operations.py)

(I note some duplication of content between 1 and 2, I'll replace the SPARQL ops in 1 with Python ops.)
